fenix/plugins/fenixdependencies/src/main/java/FenixDependenciesPlugin.kt

```diff
@@ -54,14 +51,8 @@ object FenixDependencies {
     const val androidx_benchmark_macro_junit4 = "androidx.benchmark:benchmark-macro-junit4:${FenixVersions.androidx_benchmark}"
     const val androidx_core_splashscreen = "androidx.core:core-splashscreen:${FenixVersions.androidx_splash_screen}"
     const val androidx_profileinstaller = "androidx.profileinstaller:profileinstaller:${FenixVersions.androidx_profileinstaller}"
-    const val androidx_activity_ktx = "androidx.activity:activity-ktx:${FenixVersions.androidx_activity}"
     const val androidx_legacy = "androidx.legacy:legacy-support-v4:${FenixVersions.androidx_legacy}"
-    const val androidx_lifecycle_common = "androidx.lifecycle:lifecycle-common:${FenixVersions.androidx_lifecycle}"
-    const val androidx_safeargs = "androidx.navigation:navigation-safe-args-gradle-plugin:${FenixVersions.androidx_navigation}"
-    const val androidx_navigation_fragment = "androidx.navigation:navigation-fragment-ktx:${FenixVersions.androidx_navigation}"
-    const val androidx_navigation_ui = "androidx.navigation:navigation-ui:${FenixVersions.androidx_navigation}"
     const val androidx_transition = "androidx.transition:transition:${FenixVersions.androidx_transition}"
-    const val androidx_datastore = "androidx.datastore:datastore:${FenixVersions.androidx_datastore}"
```

Comment:
We're doing something very subtle here, which is that we are moving some of these Fenix only dependencies over to the DependenciesPlugin which only should contain the AC dependencies. I don't believe we want our A-C dependencies to know about any dependencies other than the ones it needs to maintain the AC is its own independent library.

Reply:
I think we should absolutely avoid moving one-off dependencies into the AC DependenciesPlugin and instead we should make FenixDependenciesPlugin dependent on DependenciesPlugin and reuse the `Version.androidx_datastore`